### PR TITLE
Added MangaJar

### PR DESCRIPTION
**Pull request by AbdullahM0hamed**
Should close https://github.com/inorichi/tachiyomi-extensions/issues/2444

### CODE REVIEW

**Comment by CarlosEsco** [comment]
@as280093  can you provide icons?

**Comment by as280093** [comment]
On it

**Comment by as280093** [comment]
**Here the icons:**
[ic_launcher.zip](https://github.com/inorichi/tachiyomi-extensions/files/4454387/ic_launcher.25.zip)

**Comment by AbdullahM0hamed** [comment]
@as280093 thanks. Updated.

File: src/en/mangajar/src/eu/kanade/tachiyomi/extension/en/mangajar/MangaJar.kt [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        return GET("$baseUrl/manga?sortBy=popular&page=$page")
+    }
+
+    override fun latestUpdatesSelector() = popularMangaSelector()
+
+    override fun latestUpdatesRequest(page: Int): Request {
+        return GET("$baseUrl/manga?sortBy=last_chapter_at&page=$page")
+    }
+
+    override fun popularMangaFromElement(element: Element) = SManga.create().apply {
```
Comment (arkon) [inline src/en/mangajar/src/eu/kanade/tachiyomi/extension/en/mangajar/MangaJar.kt:45]: Grab the covers too.

Comment (AbdullahM0hamed) [reply src/en/mangajar/src/eu/kanade/tachiyomi/extension/en/mangajar/MangaJar.kt:45]: Done.

**Pull request merged**